README.md

```diff
@@ -1,5 +1,5 @@
-# Ansible VS Code Extension
-This extension adds language support for Ansible to VS Code.
+# Ansible Language Support by Red Hat
+This extension adds language support for Ansible to [Visual Studio Code](https://code.visualstudio.com/), via the [ansible-language-server](https://github.com/ansible/ansible-language-server)
```

Comment:
Could we, please, use detached references in documents? This would improve the readability of the source. And it's also possible to reuse those references if they repeat. Also, it looks like you've accidentally removed the trailing period at the end of the sentence, which is understandable with all of the inline link syntax cluttering that place.

```suggestion
This extension adds language support for Ansible to [Visual Studio Code], via the [ansible-language-server].


[ansible-language-server]: https://github.com/ansible/ansible-language-server
[Visual Studio Code]: https://code.visualstudio.com
```

Reply:
IMO attached reference is more readable and convenient from the reader perspective.

Reply:
I will go ahead and merge this PR. If we decide on a detached reference it can be done in a separate PR across the entire repository.